tests/utils/test_io_utils.py

```diff
@@ -96,8 +96,11 @@ def test_read_cameras_txt_nonexistent_file(self) -> None:
         self.assertIsNone(calibrations)
 
     def test_round_trip_images_txt(self) -> None:
-        """Starts with a pose. Writes the pose to images.txt (in a temporary directory). Then reads images.txt to recover
-        that same pose. Checks if the original wTc and recovered wTc match up."""
+        """
+        Starts with a pose. Writes the pose to images.txt (in a temporary directory).
```

Comment:
Do you mind pulling in the changes from the master branch, Kevin? I fixed the formatting for these docstrings there.